repository/src/main/java/org/apache/atlas/repository/store/graph/v2/tags/TagDAOCassandraImpl.java

```diff
@@ -229,10 +226,10 @@ private void initializeSchema(Map<String, String> replicationConfig) throws Atla
                         "asset_metadata text, " +
                         "updated_at timestamp, " +
                         "PRIMARY KEY ((source_id, tag_type_name), propagated_asset_id)" +
-                        ") WITH compaction = {'class': 'LeveledCompactionStrategy'};",
+                        ") WITH compaction = {'class': 'SizeTieredCompactionStrategy'};",
```

Comment:
Does this needs migration or alter command ?

Reply:
No, this has no impact since it's create table if not exists.
Migration anyway drops and re creates tables.